src/app.js

```diff
@@ -1,5 +1,163 @@
+  const b = parseInt(req.params.a, 10);
+  return Number.isNaN(a) || Number.isNaN(b)
+    ? res.status(400).json({ error: 'Parameters must be valid numbers.' })
+    : res.status(200).json({ result: subtract(a, b) });
+});
+app.post('/numbers/multiply', (req, res) => {
+  const { a, b } = req.body;
+  if (!a || !b) {
+    res.status(400).json({ error: 'Parameters "a" and "b" are required.' });
+    // eslint-disable-next-line no-restricted-globals
+  } else if (isNaN(a) && isNaN(b)) {
+    res.status(400).json({ error: 'Parameters "a" and "b" must be valid numbers.' });
+  }
+  res.status(200).send({ result: multiply(a, b) });
+});
+app.post('/numbers/divide', (req, res) => {
+  const { a, b } = req.body;
+  if (a === 0) {
+    res.status(200).send({ result: 0 });
+  } else if (b === 0) {
+    res.status(400).send({ error: 'Unable to divide by 0.' });
+  } else if (!a || !b) {
+    res.status(400).json({ error: 'Parameters "a" and "b" are required.' });
+    // eslint-disable-next-line no-restricted-globals
+  } else if (isNaN(a) && isNaN(b)) {
+    res.status(400).json({ error: 'Parameters "a" and "b" must be valid numbers.' });
+  }
+  res.status(200).send({ result: divide(a, b) });
+});
+app.post('/numbers/divide', (req, res) => {
+  const a = parseInt(req.params.b, 10);
+  const b = parseInt(req.params.a, 10);
+  res.status(200).send({ result: divide(a, b) });
+});
+app.post('/numbers/remainder', (req, res) => {
+  const { a, b } = req.body;
+  if (a === 0) {
+    res.status(200).send({ result: 0 });
+  } else if (b === 0) {
+    res.status(400).send({ error: 'Unable to divide by 0.' });
+  } else if (!a || !b) {
+    res.status(400).json({ error: 'Parameters "a" and "b" are required.' });
+    // eslint-disable-next-line no-restricted-globals
+  } else if (isNaN(a) && isNaN(b)) {
+    res.status(400).json({ error: 'Parameters must be valid numbers.' });
+  }
+  res.status(200).send({ result: remainder(a, b) });
+});
+
+// ********* Arrays Test ********* //
+
+app.post('/arrays/element-at-index/:index', (req, res) => {
+  const { index } = req.params;
+  const { array } = req.body;
+  res.status(200).json({ result: getNthElement(index, array) });
+});
+app.post('/arrays/to-string', (req, res) => {
+  const { array } = req.body;
+  res.status(200).json({ result: arrayToCSVString(array) });
+});
+app.post('/arrays/append', (req, res) => {
+  const { array, value } = req.body;
+  res.status(200).json({ result: addToArray2(value, array) });
+});
+app.post('/arrays/starts-with-vowel', (req, res) => {
+  const { array } = req.body;
+  res.status(200).json({ result: elementsStartingWithAVowel(array) });
+});
+app.post('/arrays/remove-element', (req, res) => {
+  const { array } = req.body;
+  const index = parseInt(req.query.index, 10);
+  if (req.query.index) {
+    res.status(200).json({ result: removeNthElement2(index, array) });
+  } else {
+    res.status(200).json({ result: removeNthElement2(0, array) });
+  }
```

Comment:
Theres a bit of unnecessary repetition here, you could update this by changing how you handle the index parameter. If its falsy, you want the index value to be 0, otherwise you want to parse it into an integer and use that. So this function can be re-written to just:

```
  const index = req.query.index ? parseInt(req.query.index, 10) : 0;
  res.status(200).json({ result: removeNthElement2(index, req.body.array) });
```